config/form_profile_mappings/21-2680.yml

```diff
@@ -0,0 +1,6 @@
+veteran:
+  fullName: [identity_information, full_name]
+  dateOfBirth: [identity_information, date_of_birth]
+  phoneNumber: [contact_information, us_phone]
+  email: [contact_information, email]
+  address: [contact_information, address]
```

Comment:
Do we need ssn?
```  dateOfBirth: [identity_information, date_of_birth]
  ssn: [identity_information, ssn]  # <-- add this
  phoneNumber: [contact_information, us_phone]

Reply:
@stiehlrod I originally had SSN but then removed since we should expect the profile to not provide this information since its sensitive PII. We should only expect this from SiP would we still need that here?